context/app/routes_main.py

```diff
@@ -31,7 +31,8 @@ def _get_client():
 
 @blueprint.route('/')
 def index():
-    return render_template('pages/index.html', types=types)
+    core_props = {'endpoints': {'esEndpoint': current_app.config['ELASTICSEARCH_ENDPOINT']}}
```

Comment:
Prefer to spell things out, and generally keep names consistent.
```suggestion
    core_props = {'endpoints': {'elasticsearchEndpoint': current_app.config['ELASTICSEARCH_ENDPOINT']}}
```

Reply:
I initially did that but I was getting a flask error similar to `RuntimeError: working outside of application context` .